manifests/e/electerm/electerm/1.20.6/electerm.electerm.installer.yaml

```diff
@@ -0,0 +1,29 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: electerm.electerm
+PackageVersion: 1.20.6
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  Scope: user
+  InstallerUrl: https://github.com/electerm/electerm/releases/download/v1.20.6/electerm-1.20.6-win-x64-installer.exe
+  InstallerSha256: D0DE77460645EBBDDC355A2F235075C09AA24E670619D3DBC6D71FAFC842F30B
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x64
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://github.com/electerm/electerm/releases/download/v1.20.6/electerm-1.20.6-win-x64-installer.exe
+  InstallerSha256: D0DE77460645EBBDDC355A2F235075C09AA24E670619D3DBC6D71FAFC842F30B
+  InstallerSwitches:
+    Custom: /ALLUSERS
+ManifestType: installer
+ManifestVersion: 1.1.0
+ReleaseDate: 2022-03-29

```

Comment:
Hmm weird how this update didn't get caught by [winget-pkgs-automation](https://github.com/vedantmgoyal2009/vedantmgoyal2009/tree/main/winget-pkgs-automation)